languages/ruby/exercises/concept/arrays/.docs/introduction.md

```diff
@@ -0,0 +1,84 @@
+In Ruby, **arrays** are ordered, integer-indexed collections of any object. Array indexing starts at `0`. A negative index is assumed to be relative to the end of the array — i.e. an index of `-1` indicates the last element of the array, `-2` is the next to last element in the array, and so on.
```

Comment:
This introduction file is on the large side. Maybe it would be possible to trim things done slightly? The best way to go about this is to:

1. Remove all knowledge that the student does not need to know to be able to solve the exercise
1. Combine examples